include/mitama/result/result.hpp

```diff
@@ -422,6 +422,13 @@ class [[nodiscard]] basic_result<_mutability, T, E,
   ///   Covert result to bool and returns true if the result is failure.
   constexpr bool operator !() const noexcept { return std::holds_alternative<failure_t<E>>(storage_); }
 
+  /// @brief
+  ///   Returns result storage.
+  std::variant<std::monostate, success_t<T>, failure_t<E>>
+  storage() const {
```

Comment:
I think `into_storage()` is better name.

Reply:
Thank you for your review. I will fix it.